helm/pinot/templates/minion-stateless/deployment.yaml

```diff
@@ -79,20 +79,37 @@ spec:
 {{- end }}  
         {{- if .Values.minionStateless.probes.livenessEnabled }}
         livenessProbe:
-          initialDelaySeconds: {{ .Values.probes.initialDelaySeconds }}
+          initialDelaySeconds: {{ .Values.minionStateless.probes.liveness.initialDelaySeconds | default .Values.probes.initialDelaySeconds }}
           periodSeconds: {{ .Values.probes.periodSeconds }}
+          failureThreshold: {{ .Values.minionStateless.probes.liveness.failureThreshold | default .Values.probes.failureThreshold }}
+          successThreshold: {{ .Values.probes.successThreshold }}
+          timeoutSeconds: {{ .Values.minionStateless.probes.liveness.timeoutSeconds | default .Values.probes.timeoutSeconds }}
           httpGet:
             path: {{ .Values.minionStateless.probes.endpoint }}
             port: {{ .Values.minionStateless.service.port }}
         {{- end }}
         {{- if .Values.minionStateless.probes.readinessEnabled }}
         readinessProbe:
-          initialDelaySeconds: {{ .Values.probes.initialDelaySeconds }}
+          initialDelaySeconds: {{ .Values.minionStateless.probes.readiness.initialDelaySeconds | default .Values.probes.initialDelaySeconds }}
           periodSeconds: {{ .Values.probes.periodSeconds }}
+          failureThreshold: {{ .Values.minionStateless.probes.readiness.failureThreshold | default .Values.probes.failureThreshold }}
+          successThreshold: {{ .Values.probes.successThreshold }}
```

Comment:
This should be 
```
{{ .Values.minionStateless.probes.readiness.successThreshold | default .Values.probes.successThreshold }}
```
Same for other components.

Reply:
I think `successThreshold` and `periodSeconds` don't need to be separate based on component. For complete customization perspective we can do the same but otherwise IMO, value is always going to be same